apps/www/app/examples/cards/components/create-account.tsx

```diff
@@ -24,11 +24,11 @@ export function DemoCreateAccount() {
       </CardHeader>
       <CardContent className="grid gap-4">
         <div className="grid grid-cols-2 gap-6">
-          <Button variant="outline">
+          <Button variant="outline" className="mb-4">
```

Comment:
I agree with the @qcho 's comment.
If the content has `gap-4` I think is not necessary